scripts/zones/Batallia_Downs/Zone.lua

```diff
@@ -26,8 +26,9 @@ local function registerRegionAroundNPC(zone, NPCID, zoneID)
 end
 
 zoneObject.onInitialize = function(zone)
-    UpdateNMSpawnPoint(ID.mob.AHTU)
-    GetMobByID(ID.mob.AHTU):setRespawnTime(math.random(900, 10800))
+    -- NM Persistence
+    xi.mob.nmTODPersistCache(zone, ID.mob.AHTU)
+    xi.mob.nmTODPersistCache(zone, ID.mob.WEEPING_WILLOW)
 
```

Comment:
I believe you are missing a Lumber_Jack TOD persist.
Also missing the todpersist inside weeping_willows lua